feature_extractor/map_helper.py

```diff
@@ -89,110 +94,163 @@ def update(self, obs, game_state: GameState, entity_helper: EntityHelper):
 
       # mark team/enemy/npc
       entity_obs = player_obs['Entity']
-      entity_positions = entity_obs[:, EntityAttr["row"]:EntityAttr["col"]+1].astype(int)
-      entity_populations = entity_obs[:, EntityAttr["population_id"]].astype(int)
+      valid_entity = entity_obs[:, EntityAttr["id"]] != 0
+      entities = entity_obs[valid_entity, EntityAttr["id"]]
+      ent_in_team = [ent in self.teammates for ent in entities]
+      ent_coords = entity_obs[valid_entity, EntityAttr["row"]:EntityAttr["col"]+1].astype(int)
```

Comment:
you can get this from entity_helper.member_location

Reply:
entity_helper seems to have only team member info, whereas the entity map produced here needs enemy and npcs' coords.